### PR TITLE
Remove incorrect difference between user-defined and default bridge

### LINKED ISSUE #8437: Difference between docker0 and user-defined network not exists?
File: [network/bridge.md](https://docs.docker.com/network/bridge/)

In this part, the article told me that if I use the default bridge network, I need to use -p flag to open container's port to another container, but if I use user-defined bridge network I do not need do that. Because the containers which connect to a user-defined bridge network open ports to each other by default. However, I had created 2 containers connected to docker0 which is the default bridge network, The command was:
`docker container run -it -d centos1 centos`
`docker container run -it -d centos2 centos`
 I found that this two containers open ports to each too.  After installed the openssh-server and openssh-clients,  I can use ssh connect to another container between them. 

Docker version was docker-ce-18.09 and OS was centos 7

### PR DESCRIPTION
**Pull request by Maximization**
### Proposed changes

This PR removes the first bullet point from the list differences between user-defined bridge networks and the default bridge network:

[User-defined bridges provide better isolation and interoperability between containerized applications](https://docs.docker.com/network/bridge/#differences-between-user-defined-bridges-and-the-default-bridge)

The default bridge network, similar to user-defined bridges, opens all ports to other containers and allows for inter-container communication by default. This is elaborated and demonstrated in #8973. Other relevant issue: #8437.

### Related issues (optional)

Closes #8437 and #8973

### CODE REVIEW

**Comment by usha-mandya** [comment]
@arkodg Could you PTAL? Thanks.

**Comment by thaJeztah** [comment]
One difference would be that; all containers without a `--network` specified default to be on the `bridge` network, which can be a risk as unrelated stacks/services/containers are then able to communicate.

Using a user-defined network provides a scoped network in which only containers attached to that network are able to communicate

File: network/bridge.md [hunk 1]
```diff
@@ -34,36 +34,16 @@ network.**
...
-  accidentally opening access to the outside world.
-
-  Imagine an application with a web front-end and a database back-end. The
-  outside world needs access to the web front-end (perhaps on port 80), but only
-  the back-end itself needs access to the database host and port. Using a
-  user-defined bridge, only the web port needs to be opened, and the database
-  application doesn't need any ports open, since the web front-end can reach it
-  over the user-defined bridge.
-
-  If you run the same application stack on the default bridge network, you need
```
Comment (arkodg) [inline network/bridge.md]: this is the only paragraph that needs to be removed, the one above seems accurate IMHO

Comment (Maximization) [reply network/bridge.md]: This is part of a list of differences between user-defined bridges and the default bridge. While the above is true for user-defined bridges, it's _not_ a difference with the default bridge network.

Therefore, I think it's misleading to present it as a difference.

**Comment by arkodg** [comment]
sg, can we also incorporate @thaJeztah's comment

**Comment by Maximization** [comment]
@arkodg Done

**Pull request merged**